contentcuration/contentcuration/frontend/shared/views/__tests__/visibilityDropdown.spec.js

```diff
@@ -82,14 +81,6 @@ describe('visibilityDropdown', () => {
       ).toEqual('disabled');
     });
   });
-  describe('visibility info modal', () => {
-    it('should open the info modal when button is clicked', () => {
```

Comment:
I removed this test because this behavior is the responsibility of `InfoModal` and is already tested there.